### DISCUSSION TITLE
OsuSTD Option to lower Background parallax movement

### DISCUSSION THREAD

**Discussion by PhotonTheSecond** [ppy/osu#22825] [Ideas] [answered]
I really like parallax movement in lazer so far, but when I play **cross screen jump map**, I feel like I see mini earthquake in my minitor.
Personally from a player vision, I can clearly see background shaking around from my peripheral vision( I didn't feel the shaking when I play another type of map ).

Would be nice to be able to lower the shaking down a bit.

(try focus on background while watching)

https://user-images.githubusercontent.com/114155759/224258281-efd45c5f-6c24-4269-a222-31fa1cd8cc6b.mp4

This short clip might not feel the shake that much, but the one who playing the map in full screen mode can feel the shaking more than just watching someone play the game in my opinion.

**Comment by peppy** [comment, marked as answer]
There's already an option to turn it off completely:

![osu! 2023-03-10 at 09 35 39](https://user-images.githubusercontent.com/191335/224279929-e348606c-e990-4e52-84b7-e7cd0a6f7459.png)

Would this solve your problem? Are you instead looking to keep it on but turn down the amount only?

**Reply by Glebsin** [reply]
I would like to see a slider to adjust the parallax strength in the menu and gameplay
